src/styles/annotator/components/Buckets.scss

```diff
@@ -1,48 +1,107 @@
-@use '../mixins/buttons';
-@use '../variables' as var;
-
-$bucket-bar-width: 22px;
-
-.Buckets__list {
-  background: var.$grey-2; // When sidebar is unfolded, remove the background transparency
-  pointer-events: none;
-  position: absolute;
-  height: 100%;
-  // 2020-11-20: interim and pragmatic solution for an apparent glitch on Safari and Chrome.
-  // Adding one pixel resolve this issue: https://github.com/hypothesis/client/pull/2750
-  width: $bucket-bar-width + 1;
-  left: -($bucket-bar-width);
-}
+/**
+ * These classes style buttons to appear as lozenges with an integrated arrow
+ * pointing left, up or down (right doesn't exist yet but could easily be added).
+ * *
+ * The arrow-points are created by the combination of borders and positioning.
+ * See https://css-tricks.com/snippets/css/css-triangle/ for a few examples
+ *
+ */
 
-// When the sidebar is collapsed, make the background semi-transparent so the
-// text is visible throughout (useful for pages with tight margins)
-.annotator-collapsed .Buckets__list {
-  background: rgba(0, 0, 0, 0.08);
-}
+@layer components {
```

Comment:
The styling for the buttons in the bucket bar is non-trivial, and it felt right to abstract it out into "component"s. I've done my best to boil down the styling into comprehensible classes here.